A3A/addons/core/Templates/AddonVics.hpp

```diff
@@ -11,6 +11,116 @@ class AddonVics
+        description = "Laws of war civilian vehicle pack(don't forget to enable dlc above!)";
+        loadedMessage = "Lawsofwar loaded, civilian vehicle pool expanded";
+    };
+
+    class Contact
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"A3_Soft_F_Enoch_Offroad_01"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "contact_Civ.sqf"} };
+        displayName = "Contact Vehicle Pack";
+        description = "Contact civilian vehicle pack(don't forget to enable dlc above!)";
+        loadedMessage = "Contact loaded, civilian vehicle pool expanded";
+    };
+
+    class Kart
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"A3_Soft_F_Kart_Kart_01"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "kart_Civ.sqf"} };
+        displayName = "Kart(kart)";
+        description = "Kart";
+        loadedMessage = "Kart";
+    };
+
+    class WesternSahara
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"Vehicles_F_lxWS_Offroad_01"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "westernsahara_Civ.sqf"} };
+        displayName = "Western Sahara Vehicle Pack";
+        description = "Western Sahara civilian vehicle pack(don't forget to enable cdlc above!)";
+        loadedMessage = "Western Sahara loaded, civilian vehicle pool expanded";
+    };
+
+    class ReactionForces
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"RF_Vehicles_Pickup_01"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "reactionforces_Civ.sqf"} };
+        displayName = "Reaction Forces Vehicle Pack";
+        description = "Reaction Forces civilian vehicle pack(don't forget to enable cdlc above!)";
+        loadedMessage = "Reaction Forces loaded, civilian vehicle pool expanded";
+    };
+
+    class SOG
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"wheeled_f_vietnam_02_c"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "SOG_Civ.sqf"} };
+        displayName = "S.O.G. Vehicle Pack";
+        description = "S.O.G. civilian vehicle pack(don't forget to enable cdlc above!)";
+        loadedMessage = "S.O.G. loaded, civilian vehicle pool expanded";
+    };
+
+    class GlobalMobilization
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"gm_vehicles_land_wheeled_w123_ge_civ_w123"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "gm_Civ.sqf"} };
+        displayName = "Global Mobilization Vehicle Pack";
+        description = "Global Mobilization civilian vehicle pack(don't forget to enable cdlc above!)";
+        loadedMessage = "Global Mobilization loaded, civilian vehicle pool expanded";
+    };
+
+    class CLSAIronCurtian
```

Comment:
rename to CSLAIronCurtain
```suggestion
    class CSLAIronCurtain
```